pkgs/os-specific/linux/tuxedo-rs/default.nix

```diff
@@ -1,36 +1,36 @@
-{ lib
-, fetchFromGitHub
-, rustPlatform
-}:
-let
+{ lib, fetchFromGitHub, rustPlatform, testers, tuxedo-rs }:
+rustPlatform.buildRustPackage rec {
+
+  pname = "tuxedo-rs";
+  version = "0.3.0";
 
   # NOTE: This src is shared with tailor-gui.
   # When updating, the tailor-gui.cargoDeps hash needs to be updated.
   src = fetchFromGitHub {
     owner = "AaronErhardt";
     repo = "tuxedo-rs";
-    rev = "74b863e6dcb1ec2e6c8fb02c16bb6f23b59e67f6";
-    hash = "sha256-Yujki2vGzaT8Ze5Usk8FPg8bn86MvyyPTiWuWwEw7Xs=";
+    rev = "tailor-v${version}";
+    hash = "sha256-5F9Xo+tnmYqmFiKrKMe+EEqypmG9iIvwai5yuKCm00Y=";
   };
 
-in
-rustPlatform.buildRustPackage {
-  pname = "tuxedo-rs";
-  version = "0.2.3";
-
-  inherit src;
-
   # Some of the tests are impure and rely on files in /etc/tailord
   doCheck = false;
 
-  cargoHash = "sha256-uYt442u/BIzw/lBu18LrsJf5D46oUOFzBJ5pUjCpK6w=";
+  cargoHash = "sha256-EPbh1elLOJKOrYLeBSaZ27zWGYFajiD60eFGEGaCJKw=";
+
+  passthru.tests.version = testers.testVersion {
+    package = tuxedo-rs;
+    command = "${meta.mainProgram} --version";
+    version = version;
+  };
 
   postInstall = ''
     install -Dm444 tailord/com.tux.Tailor.conf -t $out/share/dbus-1/system.d
   '';
 
   meta = with lib; {
-    description = "Rust utilities for interacting with hardware from TUXEDO Computers";
+    description =
+      "Rust utilities for interacting with hardware from TUXEDO Computers";
```

Comment:
I don't think this is an improvement to the code and a bad formatting decission by nixfmt.